RELEASE-NOTES.txt

```diff
@@ -1,8 +1,7 @@
-4.39
+4.40
 -----
 
-
-4.38
+4.39
 -----
 -   Moved edit mode into a long press action and removed the new note bar #1328
```

Comment:
This diff is confusing. 🤔 It makes it look like the "Moved edit mode..." note was part of 4.38 and now is part of 4.39. I wonder how it happened...

Anyway, the release notes files below are correct and so is the resulting file, so 🤷‍♂️

Reply:
The diff is correct. There was a mistake which ended up in double `4.38` headers and during development this second `4.38` header was used to add the release notes. So, I renamed the incorrect one and removed the empty `4.39`. Since the script added `4.40`, it resulted in this diff.